web-calculator.html

```diff
@@ -0,0 +1,20 @@
+<html>
+
+    <head>
+        <h1>
+            This calculator will add the inputs of box 1 and box 2 together.
+        </h1>
+       
+        
+        <script src='web-calculator.js'></script>
+        <link rel='stylesheet' href="web-calculator.css"></link>
+    </head>
+
+    <body>
+        <div class='sun'></div>
+        <input id='a' type='number' />
+        <input type="button" value="+"> <!--I just wanted this to be an icon but don't know how to do that.-->
```

Comment:
There are a couple of ways to do that. Could make it a `<span>` and then style it although probably prettier than this 😄 

```HTML
<span class="addButton">+</span>
```

```CSS
.addButton {
    background-color: lightgray;
    border: 1px solid black;
    border-radius: 5px;
    padding: 3px 5px;
}
```